vault/logical_system_quotas.go

```diff
@@ -92,12 +128,57 @@ from any further requests until after the 'block_interval' has elapsed.`,
 			Operations: map[logical.Operation]framework.OperationHandler{
 				logical.UpdateOperation: &framework.PathOperation{
 					Callback: b.handleRateLimitQuotasUpdate(),
+					Responses: map[int][]framework.Response{
+						http.StatusNoContent: {{
+							Description: "OK",
```

Comment:
```suggestion
							Description: http.StatusText(http.StatusNoContent),
```
This should result in "No Content" instead. Same applies to other `StatusNoContent` responses.